tests/components/sensor/test_entur_public_transport.py

```diff
@@ -1,66 +0,0 @@
-"""The tests for the entur platform."""
```

Comment:
Why remove the test?

Reply:
The test sadly tested by doing stuff too far inside the dependency, instead of mocking the dependency, thus breaking 'everything' in the test when the dependency shaked things up. Prioritized getting the functionality out over salvaging a test I poorly constructed..

Reply:
Ok